src/pages/index.js

```diff
@@ -233,7 +233,7 @@ profileEditButton.addEventListener("click", () => {
 const newCardPopupForm = new PopupWithForm(
   ".card-modal",
   handleNewCardServerRenderSubmit
-); */
```

Comment:
`      cardSection = new Section(
`

Only one instance of class `Section` should be created in the project because there is only **one** section for the cards.